lib/matplotlib/figure.py

```diff
@@ -1038,6 +1038,38 @@ def fixlist(args):
         key = fixlist(args), fixitems(kwargs.items())
         return key
 
+    def add_artist(self, a, clip=False):
+        """
+        Add any :class:`~matplotlib.artist.Artist` to the figure.
+
+        If the added artist has no transform set to it previously, its
+        transform will be set to ``transFigure``.
```

Comment:
``figure.transFigure``?

Reply:
I think ideally this docstring would be numpydoc compliant.  Most of `figure.py` is *not*, but can't hurt to make this new one compliant.